### PR TITLE
Become part of kiwi project

### PR DESCRIPTION
**Pull request by chrisrohr**
Related to #5

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -21,265 +27,56 @@
         <url>https://github.com/elucidation-project</url>
     </organization>
 
-    <developers>
```
Comment (sleberknight) [inline pom.xml:24]: Why did you remove us?

File: pom.xml [hunk 2]
```diff
@@ -21,265 +27,56 @@
...
-    </developers>
-
     <contributors>
         <contributor>
             <name>John Loizeaux</name>
             <url>https://gitlab.com/loizeaux</url>
         </contributor>
     </contributors>
 
-    <licenses>
```
Comment (sleberknight) [inline pom.xml:50]: Why removing?

File: pom.xml [hunk 3]
```diff
@@ -306,119 +103,6 @@
...
-                <version>${maven-enforcer-plugin.version}</version>
-                <executions>
-                    <execution>
-                        <id>enforce</id>
-                        <configuration>
-                            <rules>
-                                <dependencyConvergence />
-                                <bannedDependencies>
-                                    <excludes>
-                                        <!-- As recommended from slf4j guide, exclude commons-logging -->
```
Comment (sleberknight) [inline pom.xml:323]: Does kiwi-parent have these? I don't think it does, so why removing?

Comment (sleberknight) [reply pom.xml:323]: Or, do we need to _add_ them to kiwi-parent?

File: pom.xml (hunk 1 above)
Comment (chrisrohr) [reply pom.xml:24]: This will now get pulled from kiwi-parent.  Any contributors directly to this set of libraries can be added to the contributors section.

File: pom.xml (hunk 2 above)
Comment (chrisrohr) [reply pom.xml:50]: This will now get pulled from kiwi-parent.

File: pom.xml (hunk 3 above)
Comment (chrisrohr) [reply pom.xml:323]: Well I guess that is a good question.  Why aren't we banning those?

File: pom.xml [hunk 4]
```diff
@@ -306,119 +103,6 @@
...
-                                    <name>implSpec</name>
-                                    <placement>a</placement>
-                                    <head>Implementation Requirements:</head>
-                                </tag>
-                                <tag>
-                                    <name>implNote</name>
-                                    <placement>a</placement>
-                                    <head>Implementation Note:</head>
-                                </tag>
-                                <tag><name>param</name></tag>
```
[the remark this answers is not in the document]
Comment (chrisrohr) [reply pom.xml:395]: I wonder if at some point we thought if we overrode them, we had to add them all back in.

File: pom.xml (hunk 3 above)
Comment (sleberknight) [reply pom.xml:323]: So, actually in our base parent POM for services, we _do_ ban them. I am not sure why we didn't in kiwi.

File: pom.xml (hunk 4 above)
[the remark this answers is not in the document]
Comment (sleberknight) [reply pom.xml:395]: Ok, it seems to work fine without having all of them.

File: pom.xml (hunk 3 above)
Comment (sleberknight) [reply pom.xml:323]: I'm going to peruse the SLF4J guide since that first comment mentions it.

Comment (sleberknight) [reply pom.xml:323]: I don't see anything in the SLF4J docs about this, so maybe it used to be there years ago but now is not?

Comment (sleberknight) [reply pom.xml:323]: For now, let's leave this alone, but let's create a new discussion or investigation issue to see whether we should add these.

I think maybe kiwi doesn't have it, because while we use the SLF4J _API_ in our code, we don't enforce a specific logging back-end, so you could use Logback, Log4j2, JUL, etc.

But in our Dropwizard services, we enforce those bans because Dropwizard uses Logback, and thus we definitely do _not_ want any SLF4J bindings to other back-ends, as they could potentially cause fun things like infinite delegation loops (e.g. see [Delegation Loop](https://www.slf4j.org/codes.html#jclDelegationLoop)) - at least until you get a `StackOverflowError`...

Comment (sleberknight) [reply pom.xml:323]: Created #32

File: pom.xml [hunk 5]
```diff
@@ -21,265 +27,56 @@
...
-        </snapshotRepository>
-        <repository>
-            <id>sonatype-nexus-staging</id>
-            <name>Nexus Release Repository</name>
-            <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
-        </repository>
-    </distributionManagement>
-
     <modules>
-        <module>elucidation-bom</module>
```
Comment (sleberknight) [inline pom.xml:79]: I missed during my initial review that this is going away entirely. Might it be better to keep elucidation BOM, but have it import kiwi-bom as the only thing in it right now? This would allow new dependencies to be added and controlled which are specific to elucidation. Thoughts?

**Review by sleberknight — APPROVED** [review]
